tools/bin/release_version_octavia.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+. tools/lib/lib.sh
+
+if [[ -z "${DOCKER_PASSWORD}" ]]; then
+  echo 'DOCKER_PASSWORD for airbytebot not set.';
+  exit 1;
+fi
+
+docker login -u airbytebot -p "${DOCKER_PASSWORD}"
```

Comment:
nit: again for noobs and clarity: long flags in shell are intended for scripts and short flags are intended for interactive shells.  to save future users from looking what the short flags are:
```suggestion
docker login --username airbytebot --password "${DOCKER_PASSWORD}"
```